gbi.h

```diff
@@ -1003,29 +1004,29 @@ longer a multiple of 8 (DMA word). This was not used in any command anyway. */
  * Vertex (set up for use with colors)
  */
 typedef struct {
-    short          ob[3];   /** x, y, z */
-    unsigned short flag;    /** Holds packed normals, or unused */
-    short          tc[2];   /** texture coord */
-    unsigned char  cn[4];   /** color & alpha */
+    short          ob[3];   /* x, y, z */
+    unsigned short flag;    /* Holds packed normals, or unused */
+    short          tc[2];   /* texture coord */
+    unsigned char  cn[4];   /* color & alpha */
```

Comment:
I apologize, one more thing. The PR changing the comments to Doxygen style went in in the meantime. Can you remove the changes which revert them to non-Doxygen style like this one, and on the PointLight_t struct add the extra asterisks?